test/RibbonThetaVault.js

```diff
@@ -2426,6 +2426,98 @@ function behavesLikeRibbonOptionsVault(params) {
       });
     });
 
+    describe("#withdrawLater", () => {
+      time.revertToSnapshotAfterEach();
+
+      it("rejects a withdrawLater of 0 shares", async function () {
+        await expect(
+          this.vault.withdrawLater(BigNumber.from("0"))
+        ).to.be.revertedWith("!shares");
+      });
+
+      it("rejects a completeScheduledWithdrawal if nothing scheduled", async function () {
+        const depositAmount = BigNumber.from("100000000000");
+        await depositIntoVault(
+          params.collateralAsset,
+          this.vault,
+          depositAmount
+        );
+
+        await expect(
+          this.vault.completeScheduledWithdrawal()
+        ).to.be.revertedWith("Scheduled withdrawal not found");
+      });
+
+      it("rejects a scheduled withdrawal when greater than balance", async function () {
+        const depositAmount = BigNumber.from("100000000000");
+        await depositIntoVault(
+          params.collateralAsset,
+          this.vault,
+          depositAmount
+        );
+
+        //way too much
+        await expect(
+          this.vault.withdrawLater(BigNumber.from("200000000000"))
+        ).to.be.revertedWith("Insufficient shares");
+
+        // barely too much
+        await expect(
+          this.vault.withdrawLater(BigNumber.from("100000000001"))
+        ).to.be.revertedWith("Insufficient shares");
+      });
+
+      it("accepts a withdrawLater if less than or equal to balance", async function () {
```

Comment:
Let's add another test case to test if it reverts when the user does not have enough shares when they call `completeScheduledWithdrawal`